content/en/monitors/monitor_types/anomaly.md

```diff
@@ -113,20 +113,71 @@ For detailed instructions on the **Say what's happening** and **Notify your team
+
+**Note**: Anomaly detection monitors are only available to enterprise-level customers. Pro-level customers interested in anomaly detection monitors should reach out to their customer success representative or email the [Datadog billing team][11].
+
+Anomaly monitors are managed via the [same API][12] as other monitors. However, there are some fields in the request that are unique for anomaly monitors and are discussed in more detail below.
+
+### `query`
+
+The `query` property in the request body should contain a query string in the following format:
+
+```text
+avg(<query_window>):anomalies(<metric_query>, ‘<algorithm>’, <deviations>, direction=’<direction>’, alert_window=’<alert_window>’, interval=<interval>, count_default_zero=’<default_zero>’ [, seasonality=’<seasonality>’]) >= <threshold>
+```
+
+* `query_window`: a timeframe like `last_4h` or `last_7d`; the time window displayed in graphs in notifications; must be at least as large as the `alert_window` and is recommended to be around 5 times the `alert_window`
+* `metric_query`: a standard Datadog metric query (e.g., `sum:trace.flask.request.hits{service:web-app}.as_count()`)
+* `algorithm`: `basic`, `agile`, or `robust`
+* `deviations`: a positive number; controls the sensitivity of the anomaly detection
+* `direction`: the directionality of anomalies that should trigger an alert; `above`, `below`, or `both`
+* `alert_window`: the timeframe which will be checked for anomalies (e.g., `last_5m`, `last_1h`)
+* `interval`: a positive integer representing the number of seconds in the rollup interval; we recommend that the `interval` be at least a fifth of the `alert_window` duration
+* `default_zero`: use `true` for most monitors; only set to `false` if submitting a count metric in which the lack of a value should _not_ be interpreted as a zero
+* `seasonality`: `hourly`, `daily`, or `weekly`; exclude this parameter when using the `basic` algorithm
+* `threshold`: a positive number no larger than 1; the fraction of points in the `alert_window` that must be anomalous in order for a critical alert to trigger
 
 Below is an example query for an anomaly detection monitor, which alerts when the average Cassandra node's CPU is three standard deviations above the ordinary value over the last 5 minutes:
 
 ```text
 avg(last_1h):anomalies(avg:system.cpu.system{name:cassandra}, 'basic', 3, direction='above', alert_window='last_5m', interval=20, count_default_zero='true') >= 1
 ```
 
-**Note**: Anomaly detection monitors are only available to enterprise-level customers. Pro-level customers interested in anomaly detection monitors should reach out to their customer success representative or email the [Datadog billing team][11].
+### `options`
+
+Most of the properties in `options` property in the request body are the same as for other query alerts, but special attention should be given to `thresholds` and `threshold_windows`.
+
+#### `thresholds`
```

Comment:
hmm. Due to the header 4 styling, this shows up as `THRESHOLDS`. Would it be better to not make this a heading or make it header 5?

Reply:
yeah, good catch, made a suggestion to make it bold instead of h4